lib/test/detect_unit.c

```diff
@@ -87,7 +87,7 @@ main(void)
 
     if ((sum = CU_get_run_summary()) == NULL ||
         sum->nSuitesFailed || sum->nTestsFailed || sum->nAssertsFailed) {
-
```

Comment:
This line removal decreases readability of the code as the next and the previous lines has the same indentation. Please just don't remove it.